### ISSUE TITLE
TOOLS/PERF: rocm perf modules not available

### ISSUE THREAD

**Issue by paklui** [openucx/ucx#4433]
The ucx_perftest for rocm cannot run because the perf modules for rocm are not available.

**Comment by paklui** [comment]
The proposed changes in PR #4434 should provide the ucx_perftest for rocm and rocm-managed.
```
paklui@ixt-hq-50:~/ucx$ 
        HIP_VISIBLE_DEVICES=0 /home/paklui/ucx/install/bin/ucx_perftest -m rocm -t tag_bw &
        HIP_VISIBLE_DEVICES=1 /home/paklui/ucx/install/bin/ucx_perftest -m rocm -t tag_bw -s 4194304 localhost 2>&1 | tee -a $LOG
[1573535643.539054] [ixt-hq-50:18615:0]       perftest.c:1394 UCX  WARN  CPU affinity is not set (bound to 56 cpus). Performance may be impacted.
Waiting for connection...
+------------------------------------------------------------------------------------------+
| API:          protocol layer                                                             |
| Test:         tag match bandwidth                                                        |
| Data layout:  (automatic)                                                                |
| Message size: 4194304                                                                    |
+------------------------------------------------------------------------------------------+
[1573535643.539053] [ixt-hq-50:18616:0]       perftest.c:1394 UCX  WARN  CPU affinity is not set (bound to 56 cpus). Performance may be impacted.
+--------------+-----------------------------+---------------------+-----------------------+
|              |       latency (usec)        |   bandwidth (MB/s)  |  message rate (msg/s) |
+--------------+---------+---------+---------+----------+----------+-----------+-----------+
| # iterations | typical | average | overall |  average |  overall |   average |   overall |
+--------------+---------+---------+---------+----------+----------+-----------+-----------+
          6828   146.402   146.472   146.472   27309.02   27309.02        6827        6827
         13655   146.386   146.478   146.475   27307.80   27308.41        6827        6827
...
        996892   146.375   146.460   146.464   27311.26   27310.42        6828        6828
       1000000   146.369   146.512   146.464   27301.54   27310.39        6825        6828
paklui@ixt-hq-50:~/ucx$
```

**Comment by paklui** [comment]
It is fixed in #4587
